### PR TITLE
Add resellers-customers, localization and guidance text

### PR DESCRIPTION
**Pull request by TanmayJain92**
## Purpose
1. Added resellers and their customers organization flow.
2. Added localization in the app.
3. Added battery transmission and measurement guidance text while creating shipments.

### CODE REVIEW

File: src/react-query/mutations/authUser/updateUserMutation.js [hunk 1]
```diff
@@ -31,6 +31,7 @@ export const useUpdateUserMutation = (
   {
     onSuccess: () => {
       if (history) {
+        window.location.reload();
```
Comment (patelradhika) [inline src/react-query/mutations/authUser/updateUserMutation.js:34]: Why location reload on update user? Can't you just invalidate the queries and fetch again?

File: src/index.js [hunk 2]
```diff
@@ -1,7 +1,6 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import App from './App';
-import './i18n';
```
Comment (patelradhika) [inline src/index.js:4]: If you are removing i18n remove all related files that are not used anywhere

**Review by patelradhika — CHANGES_REQUESTED** [review]

**Comment by patelradhika** [comment]
image

Why is there a line coming at the bottom? Please fix this

File: src/react-query/mutations/authUser/updateUserMutation.js (hunk 1 above)
Comment (TanmayJain92) [reply src/react-query/mutations/authUser/updateUserMutation.js:34]: This is for applying localization. Also window will reload only when I'm passing the user_language.

File: src/index.js (hunk 2 above)
Comment (TanmayJain92) [reply src/index.js:4]: Done

**Review by patelradhika — APPROVED** [review]

**Pull request merged**